src/main/kotlin/de/bund/digitalservice/useid/timebasedtokens/TimeBasedTokenController.kt

```diff
@@ -24,6 +25,7 @@ import java.util.UUID
 @ConditionalOnProperty(name = ["features.desktop-solution-enabled"], havingValue = "true")
 class TimeBasedTokenController(private val timeBasedTokenService: TimeBasedTokenService) {
     @PostMapping("/{useIdSessionId}/tokens", produces = [MediaType.APPLICATION_JSON_VALUE])
+    @Operation(summary = "Create (and return) time-based token for this session")
```

Comment:
Return type makes it clear enough? And returning the created object is best practice in REST, isn't it? 
This comment is really useless. It's so unimportant but now I already wrote it. 
```suggestion
    @Operation(summary = "Create time-based token for this session")
```